### ISSUE TITLE
Implement `eventgrid` command

### ISSUE THREAD

**Issue by marstr** [Azure/buffalo-azure#6]
The `eventgrid` command extends [`BuffaloCommand:"generate"`](https://godoc.org/github.com/gobuffalo/buffalo/plugins). 

When it is executed it should:
1. Update the `app.go` file to include an action for receiving Event Grid events.
2. Add a file which will contain methods to be implemented by the Buffalo user.
    a. Without specifying event type information, there should be a single handler registered to `"all"` which receives an [eventgrid.Event](https://godoc.org/github.com/Azure/azure-sdk-for-go/services/eventgrid/2018-01-01/eventgrid#Event). 
Signature: `func WellChosenName(c buffalo.Context, e eventgrid.Event) error`
    b. ~~When type information is specified, a handler for each event type should be generated. A type should also be generated which encompasses the schema of the Event given to it.~~
Example:
``` Go
type IntEvent struct {
    eventgrid.Event `json:""`
    Data int `json:"data"`
}

func AnotherWellChosenName(c buffalo.Context, e IntEvent) error {
    // ...
}
```
3. [stretch, may be incorporated as a separate issue] If an Azure deployment template exists and a Topic ID is provided, update the template to subscribe this Buffalo App to the Topic.

[Buffalo Directory Structure for Reference](https://gobuffalo.io/en/docs/directory-structure)

edit: The strategy of generating a bunch of types was both harder for me, and not really helpful for users. Instead, `eventgrid.Event` has a `json.RawMessage` to allow partial unmarshaling by the event, and further unmarshaling by users.

**Comment by marstr** [comment]
Before this issue is closed, #16 and #17 should be completed.

**Comment by marstr** [comment]
Done 😃
